.github/workflows/bazel.yml

```diff
@@ -0,0 +1,29 @@
+name: Bazel
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: ['*']
+
+jobs:
+  MacOS:
+    strategy:
+      matrix:
+        xcode_version: ['12.5.1', '13.0', '13.2.1']
```

Comment:
Just running on the latest should be fine here. The tests are run with SwiftPM+Xcode on macOS and Linux for multiple Swift versions already.